pallets/kilt-launch/src/mock.rs

```diff
@@ -125,16 +125,12 @@ impl kilt_launch::Config for Test {
 	type PalletId = LaunchPalletId;
 }
 
-parameter_types! {
-	pub const MinVestedTransfer: Balance = MIN_VESTED_TRANSFER_AMOUNT;
-}
-
 impl pallet_vesting::Config for Test {
 	type Event = Event;
 	type Currency = Balances;
 	type BlockNumberToBalance = ConvertInto;
 	// disable vested transfers by setting min amount to max balance
-	type MinVestedTransfer = MinVestedTransfer;
+	type MinVestedTransfer = runtime_common::constants::MinVestedTransfer;
```

Comment:
I guess we want to remove this dependency here?

Reply:
yeah I skipped a few pallets because they where too much work. This seems to be one of them. 🙈

Reply:
[0007b8f](https://github.com/KILTprotocol/mashnet-node/pull/349/commits/0007b8fe13216a1e6bcddf82ff80944a5be0675e) & [4b14edb](https://github.com/KILTprotocol/mashnet-node/pull/349/commits/4b14edbdcb75bb032b59804f8b5a748eccd38c4e)
Kilt Launch & parachain staking should be the only two missing.